### PR TITLE
build: Add helm docs update to make generate command

### PR DESCRIPTION
**Pull request by terrytangyuan**
This automates helm docs generation as part of the existing `make generate` command.

### CODE REVIEW

File: charts/kserve-resources/README.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ $ helm install kserve oci://ghcr.io/kserve/charts/kserve --version v0.12.0-rc1
 | kserve.servingruntime.alibi.image | string | `"kserve/alibi-explainer"` |  |
 | kserve.servingruntime.art.defaultVersion | string | `"v0.12.0-rc1"` |  |
 | kserve.servingruntime.art.image | string | `"kserve/art-explainer"` |  |
+| kserve.servingruntime.huggingfaceserver.image | string | `"kserve/huggingfaceserver"` |  |
+| kserve.servingruntime.huggingfaceserver.tag | string | `"v0.12.0-rc1"` |  |
```
Comment (yuzisun) [inline charts/kserve-resources/README.md]: update to 0.12.0

File: charts/kserve-resources/README.md [hunk 2]
```diff
@@ -106,6 +108,3 @@ $ helm install kserve oci://ghcr.io/kserve/charts/kserve --version v0.12.0-rc1
 | kserve.storage.storageSpecSecretName | string | `"storage-config"` |  |
 | kserve.storage.tag | string | `"v0.12.0-rc1"` |  |
 | kserve.version | string | `"v0.12.0-rc1"` |  |
```
Comment (yuzisun) [inline charts/kserve-resources/README.md]: update to 0.12.0

**Comment by terrytangyuan** [comment]
Updated. 

/assign @yuzisun

**Pull request merged**